darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -1278,20 +1278,18 @@ def save_model(self, path: str) -> None:
         # TODO: the parameters are saved twice currently, once with complete
         # object, and once with PTL checkpointing.
 
-        raise_if_not(
-            path.endswith(".pth.tar"),
-            "The given path should end with '.pth.tar'.",
-            logger,
-        )
+        if path is None:
+            # default path
+            path = f"{type(self).__name__}_{datetime.now().strftime('%Y-%m-%d-%H-%M-%S')}.pt"
 
         # We save the whole object to keep track of everything
         with open(path, "wb") as f_out:
             torch.save(self, f_out)
 
         # In addition, we need to use PTL save_checkpoint() to properly save the trainer and model
         if self.trainer is not None:
-            base_path = path[:-8]
-            path_ptl_ckpt = base_path + "_ptl-ckpt.pth.tar"
+            base_path = path.split(".")[0]
```

Comment:
Shouldn't you set aside the last token instead? I.e. make it work if the provided path is e.g. `my.model.v.0.pt`